src/index.ts

```diff
@@ -99,8 +99,8 @@ export default class ImageTool implements BlockTool {
       endpoints: config.endpoints,
       additionalRequestData: config.additionalRequestData,
       additionalRequestHeaders: config.additionalRequestHeaders,
-      field: config.field,
-      types: config.types,
+      field: config.field || 'image',
```

Comment:
you already have it in `uploader.ts`. Why it is not working?

<img width="662" alt="image" src="https://github.com/user-attachments/assets/ca8e4cd3-196d-4df5-85a7-b61bfa962bc6">